src/main/target/FLYWOOF405S_AIO/target.c

```diff
@@ -20,17 +20,21 @@
 #include "drivers/io.h"
 #include "drivers/pwm_mapping.h"
 #include "drivers/timer.h"
+#ifdef FLYWOOF405S_AIO_PINIO
+#include "drivers/pinio.h"
+#endif
 
 timerHardware_t timerHardware[] = {
-    DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MC_MOTOR  | TIM_USE_FW_MOTOR,   0, 0), // S1 D(1,0,2)
-    DEF_TIM(TIM2,  CH3, PA2,  TIM_USE_MC_MOTOR  | TIM_USE_FW_SERVO,   0, 0), // S2 D(1,2,5)
-    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MC_MOTOR  | TIM_USE_FW_MOTOR,   0, 0), // S3 D(1,3,2)
-    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MC_MOTOR  | TIM_USE_FW_SERVO,   0, 1), // S4 D(1,7,5)
-    DEF_TIM(TIM3,  CH2, PB5,  TIM_USE_MC_MOTOR  | TIM_USE_FW_SERVO,   0, 0), // S5 D(2,4,7)
-    DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_MC_MOTOR  | TIM_USE_FW_SERVO,   0, 0), // S6
-    DEF_TIM(TIM8,  CH3, PC8,  TIM_USE_MC_SERVO  | TIM_USE_FW_SERVO,   0, 0), // S7
-
+    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_OUTPUT_AUTO,   0, 1), // S1
```

Comment:
this is changing the orders of the pins.

Reply:
Flywoo changed the motor order on the "FLYWOOF405S_AIO" FC compared to their previous "FLYWOOF405NANO" target, hence my need to create a new target for the former.

Reply:
You are breaking an official target. You are not just adding an unofficial target. 

Please fix this or the or will be rejected.

Reply:
It's tagged as SKIP_RELEASES. My understanding is this means it's not an official target.
![image](https://github.com/user-attachments/assets/bd98f45e-c3fd-47f9-9397-01ddc605c856)

Reply:
You are changing files used by an official target.

Reply:
Marking the official target, that was created with hardware from the manufacturer unofficial, or changing the output order of said official target is not acceptable. 

If it has a different output order, please treat it as a brand new target, in its own folder and with a name that does not confuse users of the official target.

Reply:
Firstly, can you please clarify if FLYWOOF405S_AIO is an official target? I created this unofficial target because none of you devs had the hardware from the manufacturer.

If FLYWOOF405S_AIO is an official target, then I will create a new target and folder for FLYWOOF405S_AIO_PINIO.